### DISCUSSION TITLE
Table Export

### DISCUSSION THREAD

**Discussion by Cooper-JTS** [Budibase/budibase#13511] [Help]
I am fairly new to Budibase and am having an issue that I feel like should be easy to accomplish. 

Problem: I have a table component with some data in my app that I am wanting to be able to export the data to CSV.

All of the documentation and discussions that I have seen have said to make sure that the table has the row selector turned on in the settings so that you can select all rows then use a button with an on click action to export data for the table. However, since the tables/grids got a makeover, I don't see that setting anymore. I would ideally be using a query for the data in the table.

On a side note: Have we gotten the default column widths figured out on the new table component?

**Comment by ConorWebb96** [comment]
Hey @Cooper-JTS,

You will be able to export table data once this [PR](https://github.com/Budibase/budibase/pull/13426) is merged in and released.

You can drag the column widths of the table to increase or decrease the width.

**Comment by Cooper-JTS** [comment]
@ConorWebb96

Thank you for the info on the table export. 

On the column widths, I understand that you can do that in the preview/published app, but we used to be able to assign a width in the builder to each column so that we could make the table fit the data a bit better. Is that something that will be added back?

**Reply by ConorWebb96** [reply]
Hey @Cooper-JTS,

Whatever widths you set the columns to within the data area will be pulled across to the design area. Views etc can also have their own ordering and sizes so theres more customisation available there.

I've mentioned to the engineering team about having another setting for initial default widths to be set in the design area. This could potentially get added in the previous PR I referenced.
